src/Components/Calculator.js

```diff
@@ -1,13 +1,47 @@
-// import React from 'react';
-import Child from './ChildComponent';
+import React, { useState } from 'react';
+// import Child from './ChildComponent';
```

Comment:
- [ ] [OPTIONAL] Please don't comment out unused code into the code base. Commenting allows developers to add more context to their code. But it's also a tool that is easy to misuse. Lousy commenting practices often lead to code that becomes too confusing or even misleading in the long run. Commented-out code is a familiar culprit that often appears under this category.